components/brave_wallet/browser/ethereum_permission_utils.cc

```diff
@@ -135,6 +136,11 @@ bool GetSubRequestOrigin(const GURL& old_origin,
   return AddAccountToHost(old_origin, account, new_origin);
 }
 
+std::string eTLDPlusOne(const GURL& url) {
```

Comment:
I think this one don't belong to ethereum_permission_utils.cc but belong to something more general, like brave_wallet_utils.cc.

Reply:
fixed in e926740044094f8d400cdabeb8ff4454b6fbb364